cluster/manifests/skipper/deployment.yaml

```diff
@@ -51,6 +51,7 @@ spec:
 {{- end }}
       priorityClassName: "{{ .Cluster.ConfigItems.system_priority_class }}"
       serviceAccountName: skipper-ingress
+      terminationGracePeriodSeconds: "{{ .Cluster.ConfigItems.skipper_termination_grace_period }}"
```

Comment:
Has to be unquoted here, though, otherwise it'll fail to apply.

Reply:
yes also checked the logs, thanks :)